roles/opensrp/tasks/main.yml

```diff
@@ -30,6 +30,15 @@
     mode: 0755
   with_items: "{{ opensrp_resource_sub_dirs }}"
 
+- name: Create OpenSRP multimedia directory
```

Comment:
@vincent-karuri maybe we should add a comment here that it was intentional to set permissions for just the group (and why the group can also write), since we expect that, for some deployments, the tomcat_group will also be the same group OpenMRS is part of if they were to both be able to read and write into the directory.

Reply:
and that for those deployments that OpenMRS will also be writing, it's expected that the user that owns that directory is likely going to be the OpenMRS user.

Reply:
Cool